docs/config/processors/versions-data.js

```diff
@@ -12,23 +15,93 @@ module.exports = function generateVersionDocProcessor(gitData) {
   return {
     $runAfter: ['generatePagesDataProcessor'],
     $runBefore: ['rendering-docs'],
+    blacklist: ['1.3.4-build.3588'],
     $process: function(docs) {
 
-      var versionDoc = {
+      var blacklist = this.blacklist;
+      var output = exec('npm info angular versions --json', { silent: true }).stdout;
+      var versions = processAllVersionsResponse(JSON.parse(output));
+
+      docs.push({
+        docType: 'version-data',
+        id: 'version-data',
+        template: 'angular-service.template.js',
+        outputPath: 'js/version-data.js',
+        ngModuleName: 'versionData',
+        serviceName: 'NG_VERSION',
+        serviceValue: currentVersion
+      });
+
+      docs.push({
         docType: 'versions-data',
         id: 'versions-data',
-        template: 'versions-data.template.js',
+        template: 'angular-service.template.js',
         outputPath: 'js/versions-data.js',
-        currentVersion: gitData.version
-      };
+        ngModuleName: 'versionsData',
+        serviceName: 'NG_VERSIONS',
+        serviceValue: versions
+      });
+
+
+      function processAllVersionsResponse(versions) {
+
+        var latestMap = {};
+
+        versions = versions
+          .filter(function(versionStr) {
+            return blacklist.indexOf(versionStr) === -1;
+          })
+          .map(function(versionStr) {
+              var version = semver.parse(versionStr);
+              if (version) {
+                var key = version.major + '.' + version.minor;
+                var latest = latestMap[key];
+                if (!latest || version.compare(latest) > 0) {
+                  latestMap[key] = version;
+                }
+              }
+              return version;
+          })
+          .filter(function(version) {
```

Comment:
I can't reproduce this locally. It also looks like we don't include any 0.x versions at all.